test/com/gfredericks/test/chuck/clojure_test_output_test.cljc

```diff
@@ -72,7 +72,9 @@
         tc-report (edn/read-string
                     {:readers (assoc default-data-readers
                                      'error #(-> %
-                                                 (assoc ::error-tag true)))}
+                                                 (assoc ::error-tag true))
+                                     ;; Babashka outputs #object tags for namespaces, so we need to handle them
```

Comment:
@cap10morgan what is this about? If there is a difference between bb and clj I'd like to know

Reply:
@borkdude I can grab more details if you want, but the basic thing from memory was that in error stacktraces being ingested as EDN, SCI outputs frames with `#object [Sci.Namespace ...` (or similar). Whereas JVM Clojure doesn't have any tags, so this would blow up under Babashka with an error that was no reader for `#object`.